srcpkgs/redir/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'redir'
+pkgname=redir
+version=3.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool"
+short_desc="Redirect TCP connections"
+maintainer="Andrew J. Hesford <ajh@sideband.org>"
+license="GPL-2.0-only"
+homepage="https://github.com/troglobit/redir"
```

Comment:
would be nice to add `changelog="https://github.com/troglobit/redir/raw/master/ChangeLog.md"`.

Reply:
Done.

Reply:
Thanks, merged it.

I replaced the `${homepage}` variable with the full url, in the changelog field because its nice to be able to open the template and just copy the url verbatim if you work on the template etc.